modules/knative-serving-advanced-config.adoc

```diff
@@ -15,17 +14,16 @@ Do not modify any YAML contained inside the `config` field. Some of the configur
 
 image::serving-form-view.png[Create Knative Serving form in web console Administrator Perspective]
 
-[id="knative-serving-controller-custom-certs_{context}"]
 == Controller Custom Certs
```

Comment:
I know this was already here, but just wondering whether this should be sentence case, and whether certs should be spelled out?